src/libs/ascent/utils/ascent_string_utils.cpp

```diff
@@ -50,6 +50,28 @@ void split_string(const std::string &s,
 
 } // namespace detail
 
+std::vector<std::string> split(const std::string &s, char delim)
+{
+  std::vector<std::string> elems;
+  detail::split_string(s, delim, elems);
+  return elems;
+}
```

Comment:
You can use Conduit's string splitter util

https://github.com/LLNL/conduit/blob/b8f4ad0858980e430661b2d381645bd1de5c0e4b/src/libs/conduit/conduit_utils.hpp#L350